src/main/scala/it/pagopa/interop/purposeprocess/common/readmodel/ReadModelQueries.scala

```diff
@@ -26,9 +26,10 @@ object ReadModelQueries {
     states: List[PurposeVersionState],
     excludeDraft: Boolean,
     offset: Int,
-    limit: Int
+    limit: Int,
+    exactMatchOnTitle: Boolean = false
```

Comment:
this name should match the parameter it is related to, e.g.
- `exactMatchOnTitle` if the other param is `title`
- `exactMatchOnName` if the other param is `name`

Reply:
+1

Reply:
+1